CHANGELOG.rst

```diff
@@ -4,6 +4,14 @@ Dellemc.Powerscale Change Logs
 
 .. contents:: Topics
 
+v3.7.0
+======
+
+Minor Changes
+-------------
+
+- Added check and diff mode support for synciqpolicy module.
```

Comment:
Add bug fix for sync IQ policy module.
Applicable to all such instances

Reply:
done